File: GXpert/GXpert.Web/Modules/Masters/Taluka/TalukaForm.cs
using Serenity.ComponentModel;
using System;

namespace GXpert.Masters.Forms;

[FormScript("Masters.Taluka")]
[BasedOnRow(typeof(TalukaRow), CheckNames = true)]
public class TalukaForm
{
    public string Title { get; set; }
    public int StateId { get; set; }
    public int DistrictId { get; set; }
    public string ShortName { get; set; }
}

File: GXpert/GXpert.Web/Modules/ServerTypes/Masters/TalukaForm.ts
import { StringEditor, ServiceLookupEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface TalukaForm {
    Title: StringEditor;
    StateId: ServiceLookupEditor;
    DistrictId: ServiceLookupEditor;
    ShortName: StringEditor;
}

export class TalukaForm extends PrefixedContext {
    static readonly formKey = 'Masters.Taluka';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!TalukaForm.init)  {
            TalukaForm.init = true;

            var w0 = StringEditor;
            var w1 = ServiceLookupEditor;

            initFormType(TalukaForm, [
                'Title', w0,
                'StateId', w1,
                'DistrictId', w1,
                'ShortName', w0
            ]);
        }
    }
}